template.yaml

```diff
@@ -68,9 +71,32 @@ Resources:
           Type: Schedule
       Handler: main.main
       MemorySize: 256
-      Policies:
-      - AmazonDynamoDBFullAccess
+			Policies:
+			- Version: "2012-10-17"
+				Statement:
+				- Effect: Allow
+					Action:
+					- dynamodb:PutItem
+          - dynamodb:UpdateItem
+          - dynamodb:GetItem
+          - dynamodb:Scan
+          - dynamodb:Query
+          Resource: !GetAtt MongoDBAtlasTableResource.Arn
       Runtime: python3.11
       Timeout: 900
     Type: AWS::Serverless::Function
+
+	MongoDBAtlasTableResource:
+		Type: AWS::DynamoDB::Table
+		Properties:
+			TableName: !Sub ${AWS::StackName}-mongodbatlastable
+			AttributeDefinitions:
+			- AttributeName: id
+				AttributeType: S
+			KeySchema:
+			- AttributeName: id
```

Comment:
attribute definition should be same as the current table https://github.com/SumoLogic/sumologic-appclient-sdk/blob/f47a7b94ab5fa071d4a07af639382742008c28f8/sumoappclient/omnistorage/aws.py#L241

read write capacities should also be same as current configuration https://github.com/SumoLogic/sumologic-appclient-sdk/blob/f47a7b94ab5fa071d4a07af639382742008c28f8/sumoappclient/omnistorage/aws.py#L256